content/articles/image-watermarking-with-opencv-python/index.md

```diff
@@ -0,0 +1,155 @@
+<!--more-->
+We will cover as much as possible in this article to make the reader understand OpenCV, a python library used in the automation of image watermarking and in many other real-life scenarios.
+
+### Table of contents
+
+- [Prerequisites](#prerequisites).
+- [OpenCV Library](#opencv-library).
+- [Image Watermarking](#image-watermarking).
+  - [Loading Logo Image](#loading-logo-image).
+  - [Getting Images to Be Watermarked](#getting-images-to-be-watermarked).
+  - [Placing Watermark](#placing-watermark).
+- [Saving the Watermarked Images](#saving-the-watermarked-images).
+- [Conclusion](#conclusion).
+
+### Prerequisites
+To follow through this tutorial, the reader should:
+- Have a prior understanding of the basic python and OpenCV library concepts.
+- Know how to read, write and show images using OpenCV.
+- Have knowledge on how to use Pycharm IDE.
+
+Use this [page](https://www.jetbrains.com/pycharm/download/#section=windows) to download Pycharm IDE and install it on your machine.
+
+### OpenCV Library
+[OpenCV](https://www.mygreatlearning.com/blog/opencv-tutorial-in-python/) is a handy python library that is open source meant for computer vision processing functionalities as well as doing other critical operations such as tracking camera movements, extracting 3D models of any object, creating image filters and watermarks. It is a great tool for the automation of image watermarks as we will see in this tutorial.
+The library can be installed in the following ways:
+- **Install using Anaconda**
+Anaconda is a free and open-source conditional distribution of the Python and R computer languages for scientific computing, with the goal of making package management and deployment easier. It can be downloaded and installed from this [page](https://www.anaconda.com/products/individual).
+
+After successfully installing anaconda, use the anaconda prompt to install OpenCV using the command below:
+
+```bash
+conda install -c conda-forge opencv 
+```
+- **For Windows and Linux**
+You can use pip to install OpenCV on windows by typing the following command at the terminal or command line:
+
+```bash
+pip install opencv-python
+```
+- **For Mac**
+Use homebrew with the command below to install OpenCV:
+
+```bash
+brew install opencv
+``` 
+After the installation, OpenCV can be used for multiple images watermarking as it is easier to use and more versatile. This scenario will have two folders: the `flowers` folder containing the images to be watermarked and the `watermarked_images` folder to hold the watermarked images after running the python scripts successfully.  The process is as shown below: 
+
```

Comment:
Please include the screenshot of the two folders in your machine. I'm assuming you're using PyCharm, right? Also, you haven't mentioned where PyCharm is used. A reader might wonder where does it come in. Please include it in your tutorial.

Reply:
@Bayler for the PyCharm IDE have stated it so clearly on the prerequisites and where one can download and install it on their machines checkout